ctx_test.go

```diff
@@ -2761,6 +2761,75 @@ func Test_Ctx_Render_Mount(t *testing.T) {
 	utils.AssertEqual(t, "<h1>Hello a!</h1>", string(body))
 }
 
+// go test -run Test_Ctx_Render_Mount_ParentOrSubHasViews
+func Test_Ctx_Render_Mount_ParentOrSubHasViews(t *testing.T) {
+	t.Parallel()
+
+	engine := &testTemplateEngine{}
+	err := engine.Load()
+	utils.AssertEqual(t, nil, err)
+
+	engine2 := &testTemplateEngine2{}
```

Comment:
please remove the loading function in the Render function 
https://github.com/gofiber/fiber/blob/5b1885a44c059b567e2fab9104df04ec9e0065e3/internal/template/html/html.go#L186-L192